ci-automation/vendor-testing/aws.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+# Copyright (c) 2022 The Flatcar Maintainers.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -euo pipefail
+
+# Test execution script for the AWS vendor image.
+# This script is supposed to run in the mantle container.
+
+source ci-automation/vendor_test.sh
+
+board="${CIA_ARCH}-usr"
+image_name="ci-${CIA_VERNUM}"
```

Comment:
Using this as image name fails with `AMI names must be between 3 and 128 characters long, and may contain letters, numbers, '(', ')', '.', '-', '/' and '_'`

Reply:
Yeah, locally didn't test the case where image name was used. Thanks.